libs/utils/decorate.js

```diff
@@ -234,12 +244,81 @@ function getVideoAttrs(hash, dataset) {
   return `${globalAttrs} controls`;
 }
 
+export function syncPausePlayIcon(video) {
+  if (!video.getAttributeNames().includes('data-hoverplay')) {
+    const offsetFiller = video.closest('.video-holder').querySelector('.offset-filler');
+    const anchorTag = video.closest('.video-holder').querySelector('a');
+    offsetFiller?.classList.toggle('is-playing');
+    const isPlaying = offsetFiller?.classList.contains('is-playing');
+    const indexOfVideo = (anchorTag.getAttribute('video-index') === '1' && videoCounter === 1) ? '' : anchorTag.getAttribute('video-index');
+    const changedLabel = `${isPlaying ? videoLabels?.pauseMotion : videoLabels?.playMotion}`;
+    const oldLabel = `${!isPlaying ? videoLabels?.pauseMotion : videoLabels?.playMotion}`;
+    const ariaLabel = `${changedLabel} ${indexOfVideo}`.trim();
+    anchorTag?.setAttribute('aria-label', `${ariaLabel} `);
+    anchorTag?.setAttribute('aria-pressed', isPlaying ? 'true' : 'false');
+    const daaLL = anchorTag.getAttribute('daa-ll');
+    if (daaLL) {
+      anchorTag.setAttribute('daa-ll', daaLL.replace(oldLabel, changedLabel));
+    }
+  }
+}
+
+export function addAccessibilityControl(videoString, videoAttrs, indexOfVideo, tabIndex = 0) {
+  if (!videoAttrs.includes('controls')) {
+    if (videoAttrs.includes('hoverplay')) {
+      return `<a class='pause-play-wrapper video-holder' tabindex=${tabIndex}>${videoString}</a>`;
+    }
+    return `<div class='video-container video-holder'>${videoString}
+    <a class='pause-play-wrapper' role='button' tabindex=${tabIndex} aria-pressed=true video-index=${indexOfVideo}>
+    <div class='offset-filler ${videoAttrs.includes('autoplay') ? 'is-playing' : ''}'>  
+      <img class='accessibility-control pause-icon' src='https://main--federal--adobecom.hlx.page/federal/assets/svgs/accessibility-pause.svg'/>
```

Comment:
This is going to introduce problems, the URL will not be transformed to load assets from the same domain. See here - https://main--cc--adobecom.hlx.live/products/photoshop?milolibs=video-a11y - the play / pause SVGs are still loaded from hlx.page